### PR TITLE
Add section about how to add II to local project

### PR DESCRIPTION
**Pull request by frederikrothenberger**
Additionally cleans up outdated information.

### CODE REVIEW

File: demos/using-dev-build/README.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+[comment]: # This document is mainly written to be viewed on https://internetcomputer.org/docs/current/developer-docs/integrations/internet-identity/integrate-identity/
+
 # Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
+This guide shows how to integrate and test a project with Internet Identity. The complete code example can be found [here](https://github.com/dfinity/internet-identity/tree/main/demos/using-dev-build). It uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
 
 This is a standalone project that you can copy to your own project.
 
+## Prerequisites
```
Comment (nmattia) [inline demos/using-dev-build/README.md:10]: Nice! how about also adding prerequisites about what the user should know already? i.e. "this assumes you know what a [canister](https://whatisacanister.foo) is, how to use `dfx` to deploy a canister, etc,etc"

File: demos/using-dev-build/README.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
+
+[comment]: # This document is mainly written to be viewed on https://internetcomputer.org/docs/current/developer-docs/integrations/internet-identity/integrate-identity/
```
Comment (nmattia) [inline demos/using-dev-build/README.md:2]: did you mean to make an admonition?

> **Note**
> foo bar

File: demos/using-dev-build/README.md [hunk 3]
```diff
@@ -31,6 +39,28 @@ At this point, the replica (for all practical matters, a local version of the In
...
+  "wasm": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm",
+  "remote": {
+    "id": {
+      "ic": "rdmx6-jaaaa-aaaaa-aaadq-cai"
+    }
+  }
+}
+```
+
+This adds the [dev build](https://github.com/dfinity/internet-identity#flavors) of Internet Identity to your (local) project. The `remote` property makes sure, that your project will _not_ create a copy of Internet Identity on the IC when deploying to production.
```
Comment (nmattia) [inline demos/using-dev-build/README.md:58]:
```suggestion
This adds the [dev build](https://github.com/dfinity/internet-identity#flavors) of Internet Identity to your (local) project. The `remote` property makes sure that your project will _not_ create a copy of Internet Identity on the IC when deploying to production.
```

File: demos/using-dev-build/README.md [hunk 4]
```diff
@@ -31,6 +39,28 @@ At this point, the replica (for all practical matters, a local version of the In
...
+
+Using the dev build locally makes test automation easy:
```
Comment (nmattia) [inline demos/using-dev-build/README.md:60]: Should we maybe start with that? i.e. explain why we're doing it before we actually do it

File: demos/using-dev-build/README.md [hunk 5]
```diff
@@ -31,6 +39,28 @@ At this point, the replica (for all practical matters, a local version of the In
...
+* all generated captchas are just the single letter `a`
+* the WebAuthn interactions are replaced by signing with a dummy key
```
Comment (nmattia) [inline demos/using-dev-build/README.md:62]: this is probably waaay to technical for an intro; users may not have seen the captchas or know what WebAuthn is. Maybe extract this elsewhere for "more info"?

File: demos/using-dev-build/dfx.json [hunk 6]
```diff
@@ -33,8 +31,7 @@
       "__4": "(which in this case populates 'dist/', see package.json).",
       "__5": "source must be a directory, otherwise we get a cryptic error.",
       "source": ["dist/"],
-
-      "__6": "NOTE: the 'frontend' value is needed otherwise HTTP calls are not proxied; value can be anything though.",
+      "__6": "NOTE: the 'frontend' signals dfx that it is useful to print out the front-end URL of this canister when deploying. It is not strictly required.",
```
Comment (nmattia) [inline demos/using-dev-build/dfx.json:34]: does `dfx` now proxy to the canister by default?

File: demos/using-dev-build/README.md (hunk 1 above)
Comment (frederikrothenberger) [reply demos/using-dev-build/README.md:10]: Good idea! I'll add that.

File: demos/using-dev-build/README.md (hunk 2 above)
Comment (frederikrothenberger) [reply demos/using-dev-build/README.md:2]: No, this is supposedly how to do comments in markdown: https://stackoverflow.com/questions/4823468/comments-in-markdown

File: demos/using-dev-build/dfx.json (hunk 6 above)
Comment (frederikrothenberger) [reply demos/using-dev-build/dfx.json:34]: Yes.

File: demos/using-dev-build/README.md (hunk 5 above)
Comment (frederikrothenberger) [reply demos/using-dev-build/README.md]: I changed the wording. The exact changes are already explained in the linked flavors section.

File: demos/using-dev-build/README.md [hunk 7]
```diff
@@ -1,9 +1,19 @@
...
 # Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
+This guide shows how to integrate and test a project with Internet Identity. The complete code example can be found [here](https://github.com/dfinity/internet-identity/tree/main/demos/using-dev-build). It uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
 
 This is a standalone project that you can copy to your own project.
 
+## Prerequisites
+
+* [DFX](https://internetcomputer.org/docs/current/developer-docs/build/install-upgrade-remove)
```
Comment (nmattia) [inline demos/using-dev-build/README.md:12]:
```suggestion
* [dfx](https://internetcomputer.org/docs/current/developer-docs/build/install-upgrade-remove)
```

File: demos/using-dev-build/README.md [hunk 8]
```diff
@@ -31,6 +41,25 @@ At this point, the replica (for all practical matters, a local version of the In
 
 If the IC actually lets the call (request) through to the `whoami` canister, it means that everything checked out, and the `whoami` canister just responds with the information the IC adds to requests, namely your identity (principal).
 
+### Adding Internet Identity to your Local Project
+
+This section explains how to add the [dev build](https://github.com/dfinity/internet-identity#flavors) of Internet Identity to your (local) project. It is recommended to use the dev build locally because it has modifications that make test automation easy.
```
Comment (nmattia) [inline demos/using-dev-build/README.md:46]: I think mentioning the dev build here may confuse readers. How about not mentioning it at all in the main text but maybe add a note that users can skip? Most people will just not care, although I agree some will want to know more

**Review by nmattia — APPROVED** [review]

**Pull request merged**